### PR TITLE
Add instruction to open in Draw.io

### PR DESCRIPTION
**Pull request by lucas-tortora**

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@JSto91 , @Dr-Electron , whenever you've have a minute, please take a look at this quick update.

File: README.md [hunk 1]
```diff
@@ -18,3 +18,13 @@
...
+
+# Add Design Resources To [Draw.io](https://app.diagrams.net/)
+
+1. Fork this repository. 
+2. Open [Draw.io](https://app.diagrams.net/).
+3. Click **File** > **Open Library From** > **URL**.
+4. Paste https://raw.githubusercontent.com/iota-wiki/Design-Resources/main/flowchart_library/flowchart.xml. 
+5. Open the **flowchart_library** folder. 
+6. Select **flowchart.xml** file. 
+7. Create you IOTA themed graphs.
```
Comment (lucas-tortora) [inline README.md]:
```suggestion
## Add Design Resources To [Draw.io](https://app.diagrams.net/)

1. Open [Draw.io](https://app.diagrams.net/).
2. Click **File** > **Open Library From** > **URL**.
3. Paste https://raw.githubusercontent.com/iota-wiki/Design-Resources/main/flowchart_library/flowchart.xml. 
4. Create your IOTA-themed graphs. 
```

**Comment by JSto91** [comment]
Since there will be more then 1 Lib, should we change this:
``` Paste https://raw.githubusercontent.com/iota-wiki/Design-Resources/main/flowchart_library/flowchart.xml```
to
```
 Paste the RAW URL of the library, for example:
 https://raw.githubusercontent.com/iota-wiki/Design-Resources/main/flowchart_library/flowchart.xml
```

wdyt?

File: README.md [hunk 2]
```diff
@@ -18,3 +18,10 @@
   - Please follow the Design-Guideline if you want to contribute images for the Wiki
   - If you want to add new files, such as Icons etc, place them in the correct folder to keep a clean structure
   - Be aware of how you name files
+
+## Add Design Resources To [Draw.io](https://app.diagrams.net/)
+
+1. Open [Draw.io](https://app.diagrams.net/).
+2. Click **File** > **Open Library From** > **URL**.
+3. Paste https://raw.githubusercontent.com/iota-wiki/Design-Resources/main/flowchart_library/flowchart.xml.
```
Comment (lucas-tortora) [inline README.md]:
````suggestion
3. Paste the RAW URL of the library, for example:
```
https://raw.githubusercontent.com/iota-wiki/Design-Resources/main/flowchart_library/flowchart.xml
```
````

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**